generator/whitelist.ts

```diff
@@ -78,6 +78,10 @@ const whitelist: WhitelistConfig[] = [
         basePath: 'mariadb/resource-manager',
         namespace: 'Microsoft.DBforMariaDB',
     },
+    {
+        basePath: 'msi/resource-manager',
```

Comment:
We no longer use the term `msi` to describe the product. Where did this come from? Was it autogenerated?

The name for the product is "managed identities for Azure resources" with no acronym, but we can probably find a shorter term to use here.

Reply:
It's a reference to the path in the swagger specs repo, which AFAIK is owned by your team: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/msi/resource-manager

Reply:
Thanks, I'm following up with ARM API folks on changing the folder name.

Reply:
We haven't made the change regarding the path since we are waiting on our last swagger modifying PR to be merged. Would this get picked up by the pipeline as well as future swagger changes?

Reply:
Yes that's fine - we'll be notified of the change when it happens